src/util/command_line_handling.cpp

```diff
@@ -774,7 +774,7 @@ namespace hpx { namespace util
 #endif
 
         // handle setting related to schedulers
-        queuing_ = detail::handle_queuing(cfgmap, vm, "local-priority-fifo");
+        queuing_ = detail::handle_queuing(cfgmap, vm, "shared-priority");
```

Comment:
There's a good reason why this scheduler was in a branch and not being pushed into master.

Reply:
I'm not sure if I'm missing something but I agree with you. To be clear, this is temporary.

Reply:
Has the dashboard shown any interesting effects either good/bad of making this the default?

Reply:
This would also mean that the shared-priority scheduler policy needs to be built always.

Reply:
I will still change it back to the old default before merging.